src/components/App.js

```diff
@@ -1,21 +1,56 @@
 import React from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
-// eslint-disable-next-line no-unused-vars
 import calculate from '../logic/calculate';
 
-function App() {
-  return (
-    <>
-      <header className="App-header">
-        <p>
-          This is the React-Calculator
-        </p>
-      </header>
-      <Display />
-      <ButtonPanel />
-    </>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
+
+  handleClick = (buttonName) => {
+    const stateObj = this.state;
+    const output = calculate(stateObj, buttonName);
+    this.setState({
+      total: output.total ? output.total : null,
```

Comment:
[OPTIONAL] You could also try the short circuit notation instead of ternary for shorter lines.
Ex: `output.total || null,`